server/graphql/v2/object/Comment.js

```diff
@@ -0,0 +1,46 @@
+import { GraphQLInt, GraphQLString, GraphQLObjectType } from 'graphql';
+import { strip_tags } from '../../../lib/utils';
+
+import { DateString } from '../scalar/DateString';
+import { Account } from '../interface/Account';
+// import { idEncodeResolver } from '../identifiers';
+
+const Comment = new GraphQLObjectType({
+  name: 'Comment',
+  description: 'This represents an Comment',
+  fields: () => {
+    return {
+      id: {
+        type: GraphQLInt,
+        /** TODO: Uncomment after editComment mutation is migrated to v2.
+         * The string encoded id is not compatible with the edit comment mutation
+         * from v1.
+         */
+        // resolve: idEncodeResolver('comment'),
+      },
```

Comment:
`id` should be in the new form from this day. If `internal_id` is needed, we can expose it but already mark it as deprecated.

Reply:
I didn't like the idea of creating a new field deprecated by default. So, I implemented the whole comment crud in v2.